tests/OAuth2SummitTicketTypesApiTest.php

```diff
@@ -154,6 +154,37 @@ public function testGetAllowedTicketTypes(){
         return $ticket_types;
     }
 
+    public function testGetTicketTypesApplyingPromoCode(){
+        $params = [
```

Comment:
please lets create a more valuable tests:
1. lets seed pre paid tickets and create a pre paid code and check that 
2. lets create ticket types and discount code and check the result

Reply:
done